west.yml

```diff
@@ -199,6 +199,9 @@ manifest:
     - name: openthread
       revision: e2a765599aa0379d0f72f0149a9cbafec070c12d
       path: modules/lib/openthread
+    - name: picolibc
+      path: modules/lib/picolibc
+      revision: zephyr
```

Comment:
This needs to be pointing to a specific commit.

Reply:
Thanks; I was leaving it pointing to a branch so I didn't need to update zephyr when I made changes in picolibc. Now that there's a released version of picolibc with the required changes, I'll switch to a commit instead. I assume you don't want a tag name here?

Reply:
> I assume you don't want a tag name here?

While specifying a tag name should work, commit SHA would be preferred.

[We are in the process of cleaning up the upstream module repositories of unneeded branches (and tags)](https://github.com/zephyrproject-rtos/zephyr/issues/45241), so using commit SHA would be required as part of that too.

Reply:
Fixed. Points to the 1.7.7 commit sha.